src/Model/InitialModel.py

```diff
@@ -90,25 +90,27 @@ def create_initial_model():
     patient_dict_container.set("dict_uid", dict_instance_uid(dataset))
 
     # Set RTSS attributes
-    if patient_dict_container.has_modality("rtss"):
-        patient_dict_container.set("file_rtss", filepaths['rtss'])
-        patient_dict_container.set("dataset_rtss", dataset['rtss'])
-        dict_raw_contour_data, dict_numpoints = \
-            ImageLoading.get_raw_contour_data(dataset['rtss'])
-        patient_dict_container.set("raw_contour", dict_raw_contour_data)
-
+    patient_dict_container.set("file_rtss", filepaths['rtss'])
+    patient_dict_container.set("dataset_rtss", dataset['rtss'])
+    dict_raw_contour_data, dict_numpoints = \
+        ImageLoading.get_raw_contour_data(dataset['rtss'])
+    patient_dict_container.set("raw_contour", dict_raw_contour_data)
+
+    # dict_dicom_tree_rtss will be set in advance if the program generates a new rtss
```

Comment:
```suggestion
    # dict_dicom_tree_rtss will be set in advance if the program
    # generates a new rtss
```

PEP 8 compliance - comment line too long.

Reply:
fixed